app/api/chat/route.ts

```
@@ -206,7 +166,7 @@ Format your responses using Markdown:
               const isNewChat = !chatId; // No chatId means it's a new chat
 
               // Create data for upsert
-              const chatData = {
+              const chatData: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)